wrf/utils.py

```diff
@@ -1006,3 +1006,84 @@ def wrfout_seriesReader(wrf_path,wrf_file_filter,specified_heights=None):
     #print(ds_subset)
     return ds_subset
 
+
+def write_tslist_file(fname,twr_locx,twr_locy,twr_names=None,twr_abbr=None,ij_or_ll='ll'):
```

Comment:
I would prefer something like:
`write_tslist_file(fname, lat=None, lon=None, i=None, j=None, names=None, abbr=None)`
Personally, I would confuse x/y with lat/lon instead of lon/lat...

Then:
```
if (lat is not None) and (lon is not None):
    # check for input consistency here
    ij = False
elif (i is not None) and (j is not None):
    # check for consistency
    ij = True
else: 
    raise ValueError('Need to specify lat/lon or i/j')

Reply:
I agree - I was going for direct mapping (x : lon and y:lat but we say "lat/lon" and "x/y") but this does make more sense. I'll work with this next week and let you know when I finish.